exchanges/huobi/huobi_test.go

```diff
@@ -1811,14 +1811,16 @@ func TestPairFromContractExpiryCode(t *testing.T) {
 		require.True(t, ok, "%s type must be in contractExpiryNames", cType)
 		assert.Equal(t, currency.BTC, p.Base, "pair Base should be the same")
 		assert.Equal(t, exp, p.Quote, "pair Quote should be the same")
-		d, err := time.Parse("060102", p.Quote.String())
+		cst, err := time.LoadLocation("Asia/Singapore") // Huobi HQ and apparent local time for when codes become effective
+		require.NoError(t, err, "LoadLocation must not error")
+		d, err := time.ParseInLocation("060102", p.Quote.String(), cst)
```

Comment:
Going from [this](https://www.timeanddate.com/time/zones/sgt)
```suggestion
		sgt, err := time.LoadLocation("Asia/Singapore") // Huobi HQ and apparent local time for when codes become effective
		require.NoError(t, err, "LoadLocation must not error")
		d, err := time.ParseInLocation("060102", p.Quote.String(), sgt)
```

Reply:
I think when we see a mistake happened because of unnecessary naming, best not to double down on it.

Fixed https://github.com/gbjk/gocryptotrader/commit/0f841230be